core/conn/odb/src/odb.c

```diff
@@ -10183,6 +10183,7 @@ static int Oloadbuff(int eid)
                 }
                 Oi++;
             }
+            Oi = 1; //the record number need to be initialize for next loop
```

Comment:
This is fine, but it might be a bit clearer to the reader if the initialization was at the beginning of the loop, for example at line 10090.

Reply:
Thanks, I have updated the change.